security-analysis/security-analysis-api/src/main/java/com/powsybl/security/json/ConditionSerializer.java

```diff
@@ -41,12 +41,33 @@ public void serialize(Condition condition, JsonGenerator jsonGenerator, Serializ
             case AnyViolationCondition.NAME:
                 serializeFilters((AbstractFilteredCondition) condition, jsonGenerator, serializerProvider);
                 break;
+            case InjectionThresholdCondition.NAME:
+                serializeThresholdCondition((AbstractThresholdCondition) condition, jsonGenerator);
+                break;
+            case BranchThresholdCondition.NAME:
+                BranchThresholdCondition branchCondition = (BranchThresholdCondition) condition;
+                serializeThresholdCondition(branchCondition, jsonGenerator);
+                jsonGenerator.writeStringField("side", branchCondition.getSide().name());
+                break;
+            case AcDcConverterThresholdCondition.NAME:
+                AcDcConverterThresholdCondition acDcConverterCondition = (AcDcConverterThresholdCondition) condition;
+                serializeThresholdCondition(acDcConverterCondition, jsonGenerator);
+                jsonGenerator.writeBooleanField("acSide", acDcConverterCondition.isAcSide());
+                jsonGenerator.writeNumberField("terminalNumber", acDcConverterCondition.getTerminalNumber());
```

Comment:
With the `TerminalNumber` enum: `acDcConverterCondition.getTerminalNumber().getNum()`